source/elements/oneDAL/source/algorithms/decomposition/pca.rst

```diff
@@ -32,7 +32,31 @@ oneDAL specifies two methods for PCA computation:
 -----------------------
 Covariance-based method
 -----------------------
-[TBD]
+
+This method uses eigendecomposition of the covariance matrix to define the
+principal components of the dataset. Following steps are distinguished:
+
+#. Covariance computation.
```

Comment:
Remove the full stop (same for the next two lines)

Reply:
fixed